octovis/src/ViewerGui.cpp

```diff
@@ -36,12 +36,13 @@
 
 namespace octomap{
 
-ViewerGui::ViewerGui(const std::string& filename, QWidget *parent)
+ViewerGui::ViewerGui(const std::string& filename, unsigned int initDepth, QWidget *parent)
```

Comment:
This change will break existing code using the previous list of parameters (a `QWidget*` will be interpreted as `unsigned int`). Maybe put the new optional parameter at the end?